ReactWindows/ReactNative/Touch/TouchHandler.cs

```diff
@@ -252,7 +252,7 @@ private void DispatchTouchEvent(TouchEventType touchEventType, List<ReactPointer
 
             var touchEvent = new TouchEvent(touchEventType, touches, changedIndices, coalescingKey);
 
-            _view.GetReactContext()
+            _view.GetReactContext()?
```

Comment:
Is this just a race condition in case the view is being torn down at the same time a touch occurs?

Reply:
Yes. With all enable/disable stuff, it still seems hard to guarantee no last minute event comes... Perhaps a bug in xaml, I saw it happening once.

---
In reply to: [215818995](https://github.com/Microsoft/react-native-windows/pull/1958#discussion_r215818995) [](ancestors = 215818995)